### ISSUE TITLE
Accessibility problems with Insert Symbol popup

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#7088]
### System details

    RStudio Edition : Server
    RStudio Version : 1.4.453
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem
- enable RMD visual editor and open an RMD file
- click Insert / Symbol...
- attempt to use the popup via screen reader

### Describe the problem in detail
- feature not usable with screen reader, some examples:
- popup puts screen reader (JAWS on Chrome) in virtual cursor mode, and user must know to switch to forms mode
- the search box and the dropdown have no labels or other such annotations
- nothing is said when user moves around the list of symbols with the arrow keys
- the tooltip-like popup shown when pausing on an item is also not read by screen reader (and unreachable in any way via screen reader)
- when tabbing to the grid of symbols, all entries are read out by the screen reader (some of them at least, many seem to have no text to read)

<img width="430" alt="screenshot of the insert symbol popup" src="https://user-images.githubusercontent.com/10569626/84213912-bce1d100-aa76-11ea-8058-5479873e08f2.png">

### Describe the behavior you expected
- this feature should be usable via screen reader
- although the visual editor itself is not currently accessible to screen reader users, we should ensure the rest of the components meet minimum accessibility standards if only to prevent future reuse of these components from propagating bad practices and so the feature lights up if and when we fix the editor

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by gtritchie** [comment]
Moving to v1.5; the visual editor itself is not screen-reader accessible in 1.4, and "insert symbol" is only used in that context, so can wait.

**Comment by gtritchie** [comment]
This can wait until after the Electron port, moving to 1.6.
